TOC.md

```diff
@@ -250,19 +250,14 @@
           - [Pessimistic Mode](/dm/feature-shard-merge-pessimistic.md)
           - [Optimistic Mode](/dm/feature-shard-merge-optimistic.md)
         - [Migrate from MySQL Databases that Use GH-ost/PT-osc](/dm/feature-online-ddl.md)
-        - [Filter Certain Row Changes Using SQL Expressions](/dm/feature-expression-filter.md)
+        - [Filter DMLs Using SQL Expressions](/dm/feature-expression-filter.md)
       - [DM Architecture](/dm/dm-arch.md)
       - [Benchmarks](/dm/dm-benchmark-v5.4.0.md)
     - Quick Start
       - [Quick Start](/dm/quick-start-with-dm.md)
       - [Deploy a DM cluster Using TiUP](/dm/deploy-a-dm-cluster-using-tiup.md)
       - [Create a Data Source](/dm/quick-start-create-source.md)
-      - Data Migration Scenarios
-        - [Data Migration Scenario Overview](/dm/quick-create-migration-task.md)
-        - [Migrate Data from Multiple Data Sources to TiDB](/dm/usage-scenario-simple-migration.md)
-        - [Migrate Sharded Schemas and Tables to TiDB](/dm/usage-scenario-shard-merge.md)
-        - [Migrate Incremental Data to TiDB](/dm/usage-scenario-incremental-migration.md)
-        - [Migrate Tables when There Are More Columns Downstream](/dm/usage-scenario-downstream-more-columns.md)
+      - [Data Migration Scenarios](/dm/quick-create-migration-task.md)
```

Comment:
According to the Chinese PR https://github.com/pingcap/docs-cn/pull/8119, this section needs to be at the same level as Quick Start.